packages/x-components/src/x-modules/next-queries/components/next-queries.vue

```diff
@@ -67,12 +68,13 @@
     @Prop({ default: 5 })
     protected maxItemsToRender!: number;
```

Comment:
I've just realised that if we add a default value here, then we can't render all the available items for one of this suggestions components, and as we are going to want to modify this values in almost every case, I would remove the default value and make this property optional.